src/main/java/net/minestom/server/MinecraftServer.java

```diff
@@ -318,6 +320,14 @@ public static Server getServer() {
         return serverProcess.server();
     }
 
+    public static ClientHandshakePacket.ConnectionRule getConnectionRule() {
```

Comment:
this should not exist. people can use an event to prevent transfers if they want that feature.